### PR TITLE
{lib}[GCCcore/10.2.0] fosscuda v2020b, Check v0.15.2, GDRCopy v2.1, UCX v1.9.0, ...

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 2 out of 8 (8 easyconfigs in total)
alvis-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/2196617df108eefd4c719d0db6a37688 for a full test report.

File: easybuild/easyconfigs/u/UCX/UCX-1.9.0-GCCcore-10.2.0-CUDA-11.1.1.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['a7a2c8841dc0d5444088a4373dc9b9cc68dbffcd917c1eba92ca8ed8e5e635fb']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Autotools', '20200321'),
+    ('pkg-config', '0.29.2'),
+]
+
+osdependencies = [OS_PKG_OPENSSL_DEV]
```
Comment (lexming) [inline easybuild/easyconfigs/u/UCX/UCX-1.9.0-GCCcore-10.2.0-CUDA-11.1.1.eb:29]: Same fix here as in https://github.com/easybuilders/easybuild-easyconfigs/pull/11702
```suggestion
osdependencies = [OS_PKG_IBVERBS_DEV]
```

File: easybuild/easyconfigs/g/GDRCopy/GDRCopy-2.1-GCCcore-10.2.0-CUDA-11.1.1.eb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+checksums = ['cecc7dcc071107f77396f5553c9109790b6d2298ae29eb2dbbdd52b2a213e4ea']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Autotools', '20200321'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('CUDAcore', local_cudaversion, '', True),
```
Comment (lexming) [inline easybuild/easyconfigs/g/GDRCopy/GDRCopy-2.1-GCCcore-10.2.0-CUDA-11.1.1.eb:26]:
```suggestion
    ('CUDAcore', local_cudaversion, '', True),  # needed by GDRCopy binaries, which are tools for testing
```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 12 out of 18 (8 easyconfigs in total)
node155.hydra.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/71d8075c825f8d0faf99dca7d660d143 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 6 out of 8 (8 easyconfigs in total)
alvis-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/da4c542a11f645ab10a5cbeb97605aec for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2205
**FAILED**
Build succeeded for 6 out of 8 (8 easyconfigs in total)
alvis-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/e18cc349dd4bf62a7de07b782f3a61a9 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
alvis-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/6e24391a26559ecd19fcc901236673f0 for a full test report.

**Comment by boegel** [comment]
@Micket What was the issue with the earlier failing test reports?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
> @Micket What was the issue with the earlier failing test reports?

well you have the commits along the way if you are curious, basically just a bunch of stupid mistakes as some configs copied parts from fosscuda-2020a rather than fosscuda-2020b.

**Review by boegel — APPROVED** [review]
lgtm, I'll get some more test reports submitted...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (8 easyconfigs in total)
node3406.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/97c9a827a566e517106782c28b437d72 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node3111.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/0f2bf161cdf51765cef9f88a3c41a26d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (8 easyconfigs in total)
node2304.phanpy.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/eaf9b428a31bebaea8d38839b749999c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node155.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/9e016086c524492668c814df603bd384 for a full test report.

**Pull request merged**